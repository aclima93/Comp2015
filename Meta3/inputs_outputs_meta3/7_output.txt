Line 78, col 8: Incompatible type in assigment to a1 (got _real_, expected _integer_)
