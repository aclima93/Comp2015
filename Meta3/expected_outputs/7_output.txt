Line 79, col 8: Incompatible type in assigment to a1 (got _real_, expected _integer_)
